From: =?utf-8?b?Ikpvw6NvIFbDrXRvciBTLiBGIg==?=
 <48962164+jjoaovitor7@users.noreply.github.com>
Date: Mon, 29 Jul 2019 08:13:54 -0300
Subject: pt_BR translation.
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Name[pt_BR]=Navegador Tor (Configurações do Lançador)
Comment[pt_BR]=Navegador Tor (Configurações do Lançador)

(cherry picked from commit 0bd4779cc639e549cb408a6130b4022c647414c0)
---
 share/applications/torbrowser-settings.desktop | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/share/applications/torbrowser-settings.desktop b/share/applications/torbrowser-settings.desktop
index c2c18d1..4f12adb 100644
--- a/share/applications/torbrowser-settings.desktop
+++ b/share/applications/torbrowser-settings.desktop
@@ -3,6 +3,7 @@ Name=Tor Browser Launcher Settings
 Name[fr]=Tor Browser Launcher configurations
 Name[hu]=Tor-böngésző indító beállításai
 Name[nl]=Tor Browser Launcher Instellingen
+Name[pt_BR]=Navegador Tor (Configurações do Lançador)
 Name[ru]=Tor Browser (настройки запуска)
 GenericName=Tor Browser Launcher Settings
 GenericName[hu]=Tor-böngésző indító beállításai
@@ -10,6 +11,7 @@ Comment=Tor Browser Launcher Settings
 Comment[fr]=Tor Browser Launcher configurations
 Comment[hu]=Tor-böngésző indító beállításai
 Comment[nl]=Tor Browser Launcher Instellingen
+Comment[pt_BR]=Navegador Tor (Configurações do Lançador)
 Comment[ru]=Tor Browser (настройки запуска)
 Exec=torbrowser-launcher --settings
 Terminal=false
